### PR TITLE
Fix for data_after_restart

### PR DESCRIPTION
**Pull request by spronoza**
Fix for data_after_restart #3232

### CODE REVIEW

File: tests/firefox/antivirus/data_after_restart.py [hunk 1]
```diff
@@ -69,33 +60,33 @@ def run(self, firefox):
         click(Bookmarks.StarDialog.PANEL_FOLDER_DEFAULT_OPTION.similar(0.6))
```
Comment (yeyartsev) [inline tests/firefox/antivirus/data_after_restart.py]:
```suggestion
        click(Bookmarks.StarDialog.PANEL_FOLDER_DEFAULT_OPTION.similar(0.6))
        
```

File: tests/firefox/antivirus/data_after_restart.py [hunk 2]
```diff
@@ -69,33 +60,33 @@ def run(self, firefox):
...
         click(Bookmarks.StarDialog.PANEL_OPTION_BOOKMARK_TOOLBAR.similar(0.6))
```
Comment (yeyartsev) [inline tests/firefox/antivirus/data_after_restart.py]:
```suggestion
        click(Bookmarks.StarDialog.PANEL_OPTION_BOOKMARK_TOOLBAR.similar(0.6))
        
```

File: tests/firefox/antivirus/data_after_restart.py [hunk 3]
```diff
@@ -69,33 +60,33 @@ def run(self, firefox):
...
         click(Bookmarks.StarDialog.DONE)
```
Comment (yeyartsev) [inline tests/firefox/antivirus/data_after_restart.py]:
```suggestion
        click(Bookmarks.StarDialog.DONE)
        
```

File: tests/firefox/antivirus/data_after_restart.py [hunk 4]
```diff
@@ -69,33 +60,33 @@ def run(self, firefox):
...
 
         open_browser_console()
-        assert exists(browser_console_pattern, Settings.DEFAULT_SITE_LOAD_TIMEOUT), 'Browser console displayed.'
+        assert exists(browser_console_pattern, FirefoxSettings.SITE_LOAD_TIMEOUT), 'Browser console displayed.'
 
         restart_via_console()
-        assert exists(browser_console_pattern, Settings.DEFAULT_SITE_LOAD_TIMEOUT), 'Browser console reopened.'
+        assert exists(browser_console_pattern, FirefoxSettings.SITE_LOAD_TIMEOUT), 'Browser console reopened.'
 
         click(browser_console_pattern)
```
Comment (yeyartsev) [inline tests/firefox/antivirus/data_after_restart.py]:
```suggestion
        click(browser_console_pattern)
        
```

**Comment by spronoza** [comment]
> Extract variables from `exists` function calls
> Wrap all `click` calls with asserts

Done

**Pull request merged**